### PR TITLE
API Improvement for paddle.nn.functional.group_norm and paddle.nn.GroupNorm 易用性提升

### PR DESCRIPTION
**Pull request by NKNaN**
refer to:

https://github.com/PaddlePaddle/Paddle/pull/63881

### CODE REVIEW

File: docs/api/paddle/nn/GroupNorm_cn.rst [hunk 1]
```diff
@@ -17,13 +17,13 @@ GroupNorm
...
     - **bias_attr** (ParamAttr|bool，可选) - 指定偏置参数属性的对象。如果为 False，表示参数不学习。默认值为 None，表示使用默认的偏置参数属性。具体用法请参见 :ref:`cn_api_paddle_ParamAttr` 。
-    - **data_format** (string，可选) - 只支持“NCHW”(num_batches，channels，height，width)格式。默认值：“NCHW”。
+    - **data_format** (string，可选) - 支持 “NCL”，“NCHW”，“NCDHW”，“NLC”，“NHWC”，“NDHWC” 格式。默认值：“NCHW”。
     - **name** (str，可选) - 具体用法请参见 :ref:`api_guide_Name`，一般无需设置，默认值为 None。
 
 形状
 ::::::::::::
 
-    - input：形状为(批大小，通道数，\*) 的 Tensor。
+    - input：形状为 (批大小，通道数，\*) 或 (批大小，\* 通道数) 的 Tensor。
```
Comment (zhwesky2010) [inline docs/api/paddle/nn/GroupNorm_cn.rst]: (批大小，\*，通道数)

File: docs/api/paddle/nn/functional/group_norm_cn.rst [hunk 2]
```diff
@@ -10,7 +10,7 @@ group_norm
 .. math::
     y = \frac{x - E(x)}{\sqrt(Var(x)+ epsilon )} \ast weight + bias
 
-- :math:`x`: 形状为 [批大小，通道数，\*]，其中通道数必须是 ``num_groups`` 的整数倍
+- :math:`x`: 形状为 [批大小，通道数，\*] 或 [批大小，\* 通道数]，其中通道数必须是 ``num_groups`` 的整数倍
```
Comment (zhwesky2010) [inline docs/api/paddle/nn/functional/group_norm_cn.rst]: [批大小，\*，通道数]

File: docs/api/paddle/nn/GroupNorm_cn.rst (hunk 1 above)
Comment (NKNaN) [reply docs/api/paddle/nn/GroupNorm_cn.rst]: 已修改

File: docs/api/paddle/nn/functional/group_norm_cn.rst (hunk 2 above)
Comment (NKNaN) [reply docs/api/paddle/nn/functional/group_norm_cn.rst]: 已修改

**Review by zhwesky2010 — APPROVED** [review]
LGTM

**Pull request merged**